src/property.ts

```diff
@@ -70,10 +70,13 @@ class Property extends BaseProperty {
     }
 
     reference() {
-      if (this.isArray()) {
-        return this.mongoosePath.caster.options && this.mongoosePath.caster.options.ref
-      }
-      return this.mongoosePath.options && this.mongoosePath.options.ref
+      const ref = this.isArray()
+        ? this.mongoosePath.caster.options?.ref
+        : this.mongoosePath.options?.ref
```

Comment:
This looks good, but could you bump node version in our CI file? It still uses Node v10 and optional chaining support has been added in v14.

https://github.com/SoftwareBrothers/adminjs-mongoose/blob/master/.github/workflows/push.yml
(lines 17 and 44)

```
node-version: '10.x'
```
should be
```
node-version: '18.x'
```
Let's update to v18 since it's going LTS in a few days.

Reply:
will do

Reply:
@dziraf Node version bumped 10 -> 18. 

Should the GH Actions run on PR's too? I could set it up in that way